3rdparty/edbee-lib/README.md

```diff
@@ -0,0 +1,134 @@
+edbee-lib
+=========
+[![Build Status](https://travis-ci.org/edbee/edbee-lib.svg?branch=master)](https://travis-ci.org/edbee/edbee-lib)
+
+Edbee is a Qt based Editor Component. 
+It supports Multiple-carets, Textmate Scoping/Grammar and Highlighting support.
+
+The base class of edbee is a QWidget, it has been written from scratch and is not based on the Qt default editor components. 
+
+![Screenshot of the example application](http://edbee.net/images/screenshot1.png)
+
+The main website for edbee is at http://edbee.net/
+You can find the generated documentation at http://docs.edbee.net/
+
+This repository is a clean extraction of the previous edbee monolithical repository. 
+This library doesn't contain any data files. Using it should become as easy as the following example:
+
+```C++
+#include "edbee/edbee.h"
+edbee::TextEditorWidget* widget =  new edbee::TextEditorWidget();
+```
+
+Unfortunately we're not at this point yet. Currently the edbee library will not function without loading a default keymap file.
+
+
+Examples
+--------
+
+Using the component is pretty easy. First you must setup the edbee environment.
+This process is required to make the library know the location of the settings
+files:
+
+```C++
+#include "edbee/edbee.h"
+
+// get the edbee instance
+edbee::Edbee* tm = edbee::Edbee::instance();
+
+// configure your paths
+tm->setKeyMapPath( "location/of/your/keymap/");
```

Comment:
The highlighting fail came after ken updated the edbee library (and its submodules), which had an error in the project file. Before that the highlighting worked fine, even after I moved the code to mudlet.cpp